### PR TITLE
Improved Main Menu Key mappings

### PR DESCRIPTION
**Pull request by wyatt-herkamp**

### CODE REVIEW

File: menu/menu.lua [hunk 1]
```diff
@@ -38,45 +38,51 @@ menu.key_map = {
...
+        menu.load_map()
+
     end,
     up = function()
         cycleMaps.up()
     end,
     down = function()
         cycleMaps.down()
     end,
+    enter = function()
```
Comment (jaythomas) [inline menu/menu.lua]: Why not save the `if` statement above and make this `['return'] = function()`?

Comment (wyatt-herkamp) [reply menu/menu.lua]: Because I don't know Lua. I thought we have discussed this a few times. Changing though

File: menu/menu.lua [hunk 2]
```diff
@@ -38,45 +38,51 @@ menu.key_map = {
...
-        -- ...or else load a new map
-        else
-            --Entity.list = {}
-            loadMap(mapSelect)
-
-            state.paused = not state.paused
-            menu.unload()
-            love.audio.play(sounds.chirp_down)
-        end
+        menu.load_map()
```
Comment (SuperShadowPlay) [inline menu/menu.lua]: Since we have 3 ways to load a map now, we could probably get rid of the backspace altogether since it's very un-intuitive.

Comment (wyatt-herkamp) [reply menu/menu.lua]: Fixed

**Review by SuperShadowPlay — APPROVED** [review]

**Pull request merged**